webapp/src/registerLogin/Login.tsx

```diff
@@ -113,11 +113,12 @@ export default function Login() {
     }
   }, [loginInfo]);
 
-  useEffect(() => {
+  //THIS makes the web app skip the log in
```

Comment:
It should skip it if you are logged in: if you logged in and open the login page (without previous logout) you should skip login - you are still logged in

Reply:
And if your session was removed from backend you will go to the dog list and then you get 401 unathorized which should take you to the login page again removing all elements from cookies, it also have to be changed since rn it is using 403 forbidden